WordPress/Classes/Models/Post.m

```diff
@@ -41,9 +41,9 @@ - (void)didTurnIntoFault
     self.specialType = nil;
 }
 
-- (void)didSave
+- (void)willSave
 {
-    [super didSave];
+    [super willSave];
     if ([self isDeleted]) {
```

Comment:
I know this wasn't introduced here (sorry about the noise), but can we have the *existingObjectWithID* for the isDeleted check?.